docker/package/scripts/tezos-endorser-start

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/bash
+
+# SPDX-FileCopyrightText: 2020 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+set -euo pipefail
+
+# $PROTOCOL should be defined in the system unit environment
+endorser="/usr/bin/tezos-endorser-$PROTOCOL"
+
+endorser_dir="$DATA_DIR/endorser"
+
+configure_endorser() {
+    local endorser_config="$endorser_dir/config"
+    mkdir -p "$endorser_dir"
+
+    if [ ! -f "$endorser_config" ]; then
+        "$endorser" --base-dir "$endorser_dir" \
+                  --addr "$NODE_HOST" --port "$NODE_RPC_PORT" \
+                  config init --output "$endorser_config" >/dev/null 2>&1
+    else
+        "$endorser" --base-dir "$endorser_dir" \
+                  --addr "$NODE_HOST" --port "$NODE_RPC_PORT" \
+                  config update >/dev/null 2>&1
+    fi
+}
+
+launch_endorser() {
+    configure_endorser
+    exec "$endorser" --base-dir "$endorser_dir" \
+         --addr "$NODE_HOST" --port "$NODE_RPC_PORT" \
+         run "$@"
+}
+
+launch_endorser "$@"
```

Comment:
Not much of a point to define functions to call them once in the same script (unless you need the scoping)